Source/Scene/Cesium3DTileset.js

```diff
@@ -1285,11 +1336,21 @@ define([
                 // With replacement refinement, if the tile's SSE
                 // is not sufficient, its children (or ancestors) are
                 // rendered instead
+                var useChildrenBoundUnion = t._optimizations.checkChildrenWithinParent(t, true);
```

Comment:
This workflow is a bit awkward.  Can we just compute this on load and then just check a boolean property here?

Longer-term, this is the type of processing that would be in a web worker when a tile is downloaded and before it is ready to render.